hack/test_module.sh

```diff
@@ -0,0 +1,235 @@
+
+
+kubectl -n fybrik-system create configmap sample-policy --from-file=$WORKING_DIR/sample-policy.rego
+kubectl -n fybrik-system label configmap sample-policy openpolicyagent.org/policy=rego
+while [[ $(kubectl get cm sample-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
+
+
+c=0
+while [[ $(${TOOLBIN}/kubectl get cm sample-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]
+do
+    echo "waiting"
+    ((c++)) && ((c==25)) && break
+    sleep 1
+done
+
+
+# apply fybrik application for writing
+${TOOLBIN}/kubectl apply -f $WORKING_DIR/write-fybrikapplication.yaml -n fybrik-airbyte-sample
+
+c=0
+while [[ $(${TOOLBIN}/kubectl get fybrikapplication my-app-write -n fybrik-airbyte-sample -o 'jsonpath={.status.ready}') != "true" ]]
+do
+    echo "waiting"
+    ((c++)) && ((c==30)) && break
+    sleep 1
+done
+
+kubectl wait --for=condition=ready --all pod -n fybrik-blueprints --timeout=120s
+# do the writing using airbyte module
+kubectl run my-shell --image ghcr.io/fybrik/airbyte-module-client:main --image-pull-policy=Always -n default
+kubectl wait pod --for=condition=ready my-shell -n default --timeout 20m
+kubectl exec -it my-shell -n default -- python3 /root/client.py --host my-app-write-fybrik-airbyte-sample-airbyte-module.fybrik-blueprints --port 80 --asset fybrik-airbyte-sample/userdata --operation put
+
+# check the newly written dataset by executing commands using mysql client
+echo "mysql -h mysql.fybrik-airbyte-sample.svc.cluster.local -uroot -p${MYSQL_ROOT_PASSWORD} test -e \"select dob, name from demo\"" > ${tmp_dir}/mysql-command
+${TOOLBIN}/kubectl cp ${tmp_dir}/mysql-command mysql-client:/tmp/ -n fybrik-airbyte-sample
+${TOOLBIN}/kubectl exec -i mysql-client -n fybrik-airbyte-sample -- bash /tmp/mysql-command > ${tmp_dir}/out_fybrik_write.txt
+
+# apply fybrik application for reading
+${TOOLBIN}/kubectl apply -f $WORKING_DIR/read-fybrikapplication.yaml -n fybrik-airbyte-sample
+
+c=0
+while [[ $(${TOOLBIN}/kubectl get fybrikapplication my-app-read -n fybrik-airbyte-sample -o 'jsonpath={.status.ready}') != "true" ]]
+do
+    echo "waiting"
+    ((c++)) && ((c==30)) && break
+    sleep 1
+done
+
+kubectl wait --for=condition=ready --all pod -n fybrik-blueprints --timeout=120s
+# do the reading using airbyte module
+kubectl exec -it my-shell -n default -- python3 /root/client.py --host my-app-read-fybrik-airbyte-sample-airbyte-module.fybrik-blueprints --port 80 --asset fybrik-airbyte-sample/userdata > ${tmp_dir}/out_fybrik_read.txt
+
+# check that what was written using airbyte module is identical to what was read using airbyte module.
+# skip first column from what was read as its the index.
+awk '(NR>1)' ${tmp_dir}/out_fybrik_write.txt | awk {'print $1$2$3'} > ${tmp_dir}/fybrik_write.txt
```

Comment:
should there be a `comma`? something like:
```
'print $1,$2,$3'
```
?

Reply:
it works without the comma...

Reply:
Fixed. Thanks